docs/data/joy/getting-started/tutorial/tutorial.md

```diff
@@ -17,7 +17,7 @@ By the end, you should understand how to:
 
 Here's what the final product looks like—click on the "Show code" button underneath the demo to see the full source code:
 
-{{"demo": "LoginFinal.js"}}
+{{"demo": "LoginFinal.js", "iframe": true, "height": 500, "bg": "outlined"}}
```

Comment:
This demo shows a full page. I believe it makes more sense to isolate the code in an iframe.